cg/cli/upload/gens.py

```diff
@@ -0,0 +1,61 @@
+"""Module for uploading to Gens via CLI."""
+import logging
+from typing import Optional
+
+import click
+from cg.apps.gens import GensAPI
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.constants.gene_panel import GENOME_BUILD_37
+from cg.constants.housekeeper_tags import GensAnalysisTag
+from cg.models.cg_config import CGConfig
+from cg.store import Store
+from cg.store.models import Family, Sample
+from housekeeper.store.models import File, Version
+
+from cg.cli.upload.utils import suggest_cases_to_upload
+from cg.cli.workflow.commands import (
+    ARGUMENT_CASE_ID,
+    OPTION_DRY,
+)
+
+LOG = logging.getLogger(__name__)
+
+
+@click.command()
+@ARGUMENT_CASE_ID
+@OPTION_DRY
+@click.pass_obj
+def gens(context: CGConfig, case_id: Optional[str], dry_run: bool):
+    """Upload data from an analysis to Gens."""
+
+    click.echo(click.style("----------------- GENS -------------------"))
+
+    status_db: Store = context.status_db
+    housekeeper_api: HousekeeperAPI = context.housekeeper_api
+    gens_api: GensAPI = context.gens_api
+
+    gens_api.set_dry_run(dry_run=dry_run)
+
+    if not case_id:
+        suggest_cases_to_upload(status_db=status_db)
+        raise click.Abort
+
+    family: Family = status_db.family(case_id)
+
+    for sample in family.samples:
```

Comment:
I find this a bit odd.
`family.samples` says it's a `List[str]`-property, but `sample` seems to be a `Sample`.
Is it safe to fetch the sample ids like this?

Reply:
Good catch! I think you are right the type hint on ` _get_samples` and `samples` should be `List[Sample]`. Could you update?

Reply:
I ran into a problem trying to make this change.
The straight-forward solution of just changing `str` to `Sample` doesn't work perfectly as `Sample` is defined after `Family`.
Do you think any of the solutions [here](https://stackoverflow.com/questions/55320236/does-python-evaluate-type-hinting-of-a-forward-reference) would work, or should I see if it's possible to reorder the definitions?

Reply:
I'll resolve this for now so I can merge, but I'll make an issue or something to fix it.